locales/sv-SE/src/tasks.ts

```diff
@@ -2,123 +2,129 @@ import {removeUndefinedLocaleResources} from 'sanity'
+  'document.footer.open-tasks.text_other': '{{count}} öppna uppgifter',
 
-  'empty-state.list.assigned.heading': undefined, // 'You haven\'t been assigned any tasks'
-  'empty-state.list.assigned.text': undefined, // 'Once you\'re assigned tasks they\'ll show up here'
-  'empty-state.list.create-new': undefined, // 'Create new task'
-  'empty-state.list.document.heading': undefined, // 'This document doesn\'t have any tasks yet'
-  'empty-state.list.document.text': undefined, // 'Once a document has connected tasks, they will be shown here.'
-  'empty-state.list.subscribed.heading': undefined, // 'You haven\'t subscribed to any tasks'
-  'empty-state.list.subscribed.text': undefined, // 'When you create, modify, or comment on a task you will be subscribed automatically'
-  'empty-state.status.list.closed.assigned.heading': undefined, // 'No completed tasks'
-  'empty-state.status.list.closed.assigned.text': undefined, // 'Your tasks marked done will show up here'
-  'empty-state.status.list.closed.document.heading': undefined, // 'No completed tasks'
-  'empty-state.status.list.closed.subscribed.heading': undefined, // 'No completed tasks'
-  'empty-state.status.list.closed.subscribed.text': undefined, // 'Tasks you subscribe to marked done will show up here'
-  'empty-state.status.list.open.assigned.heading': undefined, // 'You\'re all caught up'
-  'empty-state.status.list.open.assigned.text': undefined, // 'New tasks assigned to you will show up here'
-  'empty-state.status.list.open.document.heading': undefined, // 'No tasks on this document'
-  'empty-state.status.list.open.subscribed.heading': undefined, // 'No subscribed tasks'
-  'empty-state.status.list.open.subscribed.text': undefined, // 'Tasks you subscribe to will show up here'
+  'empty-state.list.assigned.heading': 'Du har inte blivit tilldelad några uppgifter',
+  'empty-state.list.assigned.text': 'När du blir tilldelad uppgifter kommer de att visas här',
+  'empty-state.list.create-new': 'Skapa ny uppgift',
+  'empty-state.list.document.heading': 'Detta dokument har inte några uppgifter ännu',
+  'empty-state.list.document.text':
+    'När ett dokument har kopplade uppgifter kommer de att visas här.',
+  'empty-state.list.subscribed.heading': 'Du har inte prenumererat på några uppgifter',
+  'empty-state.list.subscribed.text':
+    'När du skapar, ändrar eller kommenterar en uppgift kommer du automatiskt att prenumerera',
+  'empty-state.status.list.closed.assigned.heading': 'Inga slutförda uppgifter',
+  'empty-state.status.list.closed.assigned.text':
+    'Dina uppgifter markerade som klara kommer att visas här',
+  'empty-state.status.list.closed.document.heading': 'Inga slutförda uppgifter',
+  'empty-state.status.list.closed.subscribed.heading': 'Inga slutförda uppgifter',
+  'empty-state.status.list.closed.subscribed.text':
+    'Uppgifter du prenumererar på markerade som klara kommer att visas här',
+  'empty-state.status.list.open.assigned.heading': 'Du är helt ikapp',
```

Comment:
Doesn't feel correct with the word "helt". And if we were to keep "Du är ikapp" then I feel like some other type of sentence would be more appropriate/descriptive. What do you think?